### DISCUSSION TITLE
GraphQL filtering by site name?

### DISCUSSION THREAD

**Discussion by llamafilm** [netbox-community/netbox#17428] [Q&A]
I'm hoping this is as simple as it sounds... I want to get a list of devices at a particular site by name.  The GraphiQL editor UI suggests that `site` is a valid filter, but it's not working.  

<img width="507" alt="image" src="https://github.com/user-attachments/assets/86eecb82-5b6b-4fad-8aec-9fa66803c0af">

This query returns all device at all sites, as if the filter is completely ignored.

If I use `site_id` in the filter, it works correctly, but I don't want to do that because I don't know the ID in this case.

**Comment by candlerb** [comment]
Netbox version?

**Reply by llamafilm** [reply]
I'm on 4.0.8
